src/client/client.c

```diff
@@ -502,6 +502,86 @@ static int method_daemon_reload_on(Client *client, char *node_name) {
         return 0;
 }
 
+int set_loglevel_by_node_name(Client *client, const char *node_name, char *loglevel) {
+
+        _cleanup_sd_bus_error_ sd_bus_error error = SD_BUS_ERROR_NULL;
+        _cleanup_sd_bus_message_ sd_bus_message *result = NULL;
+        char *path = NULL;
+        int r = assemble_object_path_string(NODE_OBJECT_PATH_PREFIX, node_name, &path);
+        if (r < 0) {
+                return r;
+        }
+
+        r = sd_bus_call_method(
+                        client->api_bus,
+                        HIRTE_INTERFACE_BASE_NAME,
+                        path,
+                        NODE_INTERFACE,
+                        "SetNodeLogLevel",
+                        &error,
+                        &result,
+                        "s",
+                        loglevel);
+        if (r < 0) {
+                return r;
+        }
+        return 0;
+}
+
+static int method_set_log_level(Client *client, UNUSED char *node_name, UNUSED char *loglevel) {
+        int r = 0;
+        _cleanup_sd_bus_error_ sd_bus_error error = SD_BUS_ERROR_NULL;
+        _cleanup_sd_bus_message_ sd_bus_message *result = NULL;
+
+        if (node_name != NULL) {
+                r = set_loglevel_by_node_name(client, node_name, loglevel);
+                if (r < 0) {
+                        return r;
+                }
+        } else {
```

Comment:
I think set-loglevel on all should be implemented on manager side, and there should be a separate method on the manager interface for that. Moving this logic to the client 1. Complicates the client 2. Makes it harder to do this via API (basically if you use the API and not the client, you should re-implement this logic yourself)

Reply:
My first try to implement this was to call the manager via sdbus and then the manager will use sdbus to call the methods for the Nodes. I got some kind of symbolic link error. If you think it's better to write the implementation in the Manager I can try and try to do it again.

Reply:
I definitely think that the client should be as thin as possible. Please do, and if you get errors I will be happy to assist with debug.
Look at `EnableMetrics`, it does something similar

Reply:
I think this is going into the wrong direction. Please don't implement any "on all nodes" functionality, but only set log level for 
- the manager (hirte)
- a specific (hirte-agent)
as described in https://github.com/containers/hirte/issues/189#issuecomment-1560632309. Its ok to have the user call `hirtectl set-loglevel <node> <level>` multiple times for each node.

Reply:
Nothing :) We just didn't discuss this option in our meeting so I didn't specify it in the issue - and to avoid changing the scope, I'd not put this in this PR. If we want to add it later, we can do so, of course.

Reply:
I was not aware of the meeting, but client implementation caught my eye.
I agree with this approach - finish up this commit with functionality you described above, and then add the set-node-loglevel all functionality in a separate commit